src/lib.rs

```diff
@@ -46,6 +46,19 @@
 //! bundle.write_to(write)?;
 //! # Result::Ok::<(), anyhow::Error>(())
 //! ```
+//! 
+//! ## 'webbundle' command line tool
+//! 
+//! ### create
+//! ```no_run
+//! $ webbundle create -b "https://example.com/" -p "https://example.com/foo/index.html" example.wbn foo
```

Comment:
It looks the [build](https://github.com/google/webbundle/pull/1/checks?check_run_id=477346221) fails.

> ---- src/lib.rs -  (line 53) stdout ----
> error: expected expression, found `$`
>  --> src/lib.rs:54:1
>   |
> 4 | $ webbundle create -b "https://example.com/" -p "https://example.com/foo/index.html" example.wbn foo
>   | ^ expected expression
> 
> error: aborting due to previous error
> 

I think "cargo test" tries to compile a code block in to doc comment as Rust code even if we mark it `no_run`.
They don't run code, but they compile it as Rust code. If the compile fails, it is considered as a test failure, I think.

My suggestion is:
Since this is a comment for the CLI tool (in main.rs), how about moving this document into README.md?
That might make sense. What do you think?

Reply:
@hayatoito Seems good idea~
Than should I move all comments in here(not only CLI part) to README?

Reply:
Ah, sorry for the lack of clarification.
I think we might want to move only CLI part.

Reply:
@hayatoito updated~please check when you have time 🙏